domenetjenester/mottak/src/main/java/no/nav/foreldrepenger/mottak/dokumentmottak/impl/Kompletthetskontroller.java

```diff
@@ -88,7 +89,10 @@ void persisterDokumentOgVurderKompletthet(Behandling behandling, MottattDokument
             if (erRegisterinnhentingPassert(behandling)) {
                 // Reposisjoner basert på grunnlagsendring i nylig mottatt dokument. Videre reposisjonering gjøres i task etter registeroppdatering
                 behandlingProsesseringTjeneste.utledDiffOgReposisjonerBehandlingVedEndringer(behandling, grunnlagSnapshot);
-                behandlingProsesseringTjeneste.tvingInnhentingRegisteropplysninger(behandling);
+                // Det kan komme flere inntektsmeldinger pr dag, trenger ikke oppdatering for hver av dem.
+                if (!DokumentTypeId.INNTEKTSMELDING.equals(mottattDokument.getDokumentType())) {
+                    behandlingProsesseringTjeneste.tvingInnhentingRegisteropplysninger(behandling);
```

Comment:
Jeg er ikke helt inne i all logikken her, men vil ikke dette innebære at man ikke henter inn registeropplysninger i det hele tatt?

Reply:
Nei, tvingInnhentingsørger bare for at det innhentes nye opplysninger selv om det allerede er innhentet samme dag.
Det trengs for fødsler, søknader, vedtak om pleiepenger, mm - ting som kommer i løpet av dagen.. Men ikke for IM

GjenopptaOppdaterFortsett vil ellers bare hente nye registerdata dersom mer enn 10 timer gamle.